src/mainsection.jsx

```diff
@@ -0,0 +1,14 @@
+export default function MainSection() {
+    return (
+        <>
+            <section className="main-section">
+                <img className="card" src="/tabernacle-ph.png"></img>
+                <input className="search-bar" type="search"></input>
```

Comment:
You should add `placeholder` text here to indicate that it's a search bar.

Example: `<input className="search-bar" type="search" placeholder="Search Card..."></input>`